### DISCUSSION TITLE
im making a prodigy chat thing

### DISCUSSION THREAD

**Discussion by ghost** [Prodigy-Hacking/ProdigyMathGameHacking#1143] [(D) Developer]
its a thing to insert a live chat into prodigy, anyone wanna help me with it

**Comment by FzHusky** [comment]
i would but idk how to code

**Reply by ghost** [reply]
hmmm are you good at anything else

**Reply by FzHusky** [reply]
im good at testing

**Reply by ghost** [reply]
hmm maybe you can help idk

**Reply by ghost** [reply]
ok sure you can help

**Reply by ghost** [reply]
go talk and give feedback and ideas and shit over here https://github.com/Prodigy-Chat/chat-server

**Comment by donaldli2020** [comment]
You'll need a database.

**Reply by ghost** [reply]
i have a database big brain i have already thought of some of this stuff

**Comment by ThatLegend** [comment]
I can maybe help

**Comment by ThatLegend** [comment]
Whats ur ideas?

**Reply by ghost** [reply]
well i sorta have a chat already but i need help like re-designing it and re-coding it its broken for some people too

**Comment by ThatLegend** [comment]
ok if you give me maintainer i can fix it.
wait nvm

**Comment by ThatLegend** [comment]
we could make an extension on google.

**Reply by ghost** [reply]
i already have done that

**Reply by ThatLegend** [reply]
whats it called

**Reply by ThatLegend** [reply]
like published and you can download it?

**Reply by ThatLegend** [reply]
oh i see it

**Reply by ghost** [reply]
its called prodigy chat, search on chrome extension store

**Reply by ThatLegend** [reply]
whats ur email ima send u somthin

**Reply by ThatLegend** [reply]
i sent u something

**Reply by ghost** [reply]
my email is dck.dachickenking@gmail.com

**Reply by ThatLegend** [reply]
kk

**Comment by ThatLegend** [comment]
thanks. get back to me when possible

**Reply by ghost** [reply]
ok i will but i gtg

**Reply by ThatLegend** [reply]
yo

**Comment by ThatLegend** [comment]
did u do it?

**Comment by ThatLegend** [comment]
btw how do i edit it?

**Reply by ghost** [reply]
im not letting you edit the extension. you havent even said what you want to change and you're a random stranger on the internet asking for permission to edit my extension

**Reply by ghost** [reply]
also you're impersonating a google employee so stop

**Reply by PatheticMustan** [reply]
con con, he's not impersonating anyone... that's not the page for a "google employee"

**Reply by ghost** [reply]
he said hes a google devops engineer

**Comment by ThatLegend** [comment]
its a program called google dev

**Reply by ghost** [reply]
you literally said you're a google devops engineer stop

**Comment by ThatLegend** [comment]
not google
